spec/unit/form_builder_spec.rb

```diff
@@ -98,20 +98,20 @@ def build_form(options = {}, form_object = Post.new, &block)
     end
 
     it "should generate a text input" do
-      expect(body).to have_selector("input[type=text][name='post[title]']")
+      expect(body).to have_field("post[title]", type: "text")
     end
 
     it "should generate a textarea" do
-      expect(body).to have_selector("textarea[name='post[body]']")
+      expect(body).to have_css("textarea[name='post[body]']")
```

Comment:
Shouldn't this also be another `have_field` matcher? From the docs it says it should work with textarea's.

Reply:
Does the `have_field` matcher not work here and in general for any other tests using textarea? The docs state that it will.

Reply:
I think it is because of `post[body]`. I've reported an issue at https://github.com/rubocop/rubocop-capybara/issues/78

Reply:
Ok great thanks!